src/app/source-file-tabs/source-file-tabs.service.ts

```diff
@@ -43,9 +43,10 @@ export class SourceFileTabsService {
     const fileTabsSchematic =
       this.descriptorLanguageService.toolDescriptorTypeEnumToExtendedDescriptorLanguageBean(descriptorLanguage).fileTabs;
 
-    // Always have the Descriptor Files Tab and Test Parameter Files tab
-    fileTabs.set(fileTabsSchematic[0].tabName, []);
-    fileTabs.set(fileTabsSchematic[1].tabName, []);
+    // Display all of the tabs, even if they are empty.
```

Comment:
Doesn't this mean there will always be displayed, for example, a configuration tab for legacy workflows and tools, and the tab will never have any files?

If so, what's the thinking? Not necessarily disagreeing at this point, just trying to understand.

Reply:
> Doesn't this mean there will always be displayed, for example, a configuration tab for legacy workflows and tools, and the tab will never have any files?
> 
> If so, what's the thinking? Not necessarily disagreeing at this point, just trying to understand.

Yes.  The rationale is that we got confused when the "Configuration File" tab was hidden, so might as well always display it.  A little more consistency, at the expense of a small increase in UI busyness.  I don't have strong feelings either way, but "display all of them always" also gets points from me because it has the simplest rules.

Reply:
Might be good to have a configuration tab for legacy anyway, maybe a follow-up ticket to advertise github apps integration there to further push folks from legacy to github apps